Documentation/gettingstarted/k8s-install-openshift-okd.rst

```diff
@@ -261,3 +261,8 @@ Delete the cluster
 .. code-block:: shell-session
 
    openshift-install destroy cluster --dir="${CLUSTER_NAME}"
+
+.. admonition:: Video
+  :class: attention
+
+  To learn more about OpenShift and Cilium, check out `eCHO episode 31: OpenShift Test Environment with Cilium <https://youtu.be/4jwIBS5IzYg?t=945>`__.
```

Comment:
Please use the full youtube.com URL for consistency with the other links.

Reply:
Implemented the fix.